editors/code/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "compilerOptions": {
         "esModuleInterop": false,
         "module": "commonjs",
+        "moduleResolution": "Node16",
```

Comment:
Should this be `node16` as in the docs?

Reply:
Ah, there is some reasons:

1. Future enhancement. This mode (`node16`) is more natural behavior with the current Node.js' package resolution than `node` (the current default value). And [package.json's `exports`](https://nodejs.org/api/packages.html#exports-sugar) would be a wide spreading. I think it's better to enable as early as possible.
2. Compatibility with `option-t` that depends on `exports` field too.

Reply:
Sorry -- I meant `node16` instead of `Node16`, with a lower-case `n`.

Reply:
(Mysteriously), it works in either. TypeScript compiler use lower-case but vscode's intellisense uses CamelCase.
I'll address it to lower-case.

Reply:
I guess either is fine then as long as nothing complains.

Reply:
I pushed the addressed commit: https://github.com/rust-lang/rust-analyzer/pull/15160/commits/f780810bce11bd219782e824709c36846da3b192